portfolio/src/main/webapp/script.js

```diff
@@ -66,8 +66,13 @@ function makeLarger(pic){
  * fetches comments from data servlet
  */
 function getComments(){
-    fetch('/data').then(response => response.json()).then((comments) => {
     const commentsContainer = document.getElementById('comments-container');
+    commentsContainer.innerHTML = "";
```

Comment:
Use single quotes for strings in JavaScript code. (Multiple instances of this throughout this file)
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes